MultiLoggerSingleton/Logger.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.IO;
+
+namespace MultiLoggerSingleton
+{
+    public class Logger: ILogger
+    {
+        public static int m_count = 0;
```

Comment:
What benefit this m_count will do?

Reply:
Added while writing code to check that constructor is getting called only once for Singleton.
I will remove it.